packages/node_modules/nav-frontend-alertstriper/src/alertstripe.tsx

```diff
@@ -7,19 +7,20 @@ import 'nav-frontend-alertstriper-style';
 
 const cls = (type, solid, className) => classNames('alertstripe', className, {
     'alertstripe--solid': solid,
```

Comment:
AlertstripeStopp har ingen solid state i Zeplin, så gjerne sørg for at denne oppfører seg likt uavhengig av solid-state, dvs. den skal ikke få `alertstripe--solid`.